lib/archethic/metrics/metric_helper_functions.ex

```diff
@@ -0,0 +1,381 @@
+defmodule ArchEthic.Metrics.MetricHelperFunctions do
```

Comment:
I think you should leverage doctests  to validate your functions, as they are business logic oriented for the metrics point of view.

Maybe rename it to: `ArchEthicWeb.Metrics.Helpers`